src/Avalonia.Themes.Fluent/Controls/DataValidationErrors.xaml

```diff
@@ -35,7 +35,7 @@
       <ItemsControl Items="{Binding}">
         <ItemsControl.ItemTemplate>
           <DataTemplate>
-            <TextBlock Text="{Binding Message}" Foreground="{DynamicResource SystemControlErrorTextForegroundBrush}" TextWrapping="Wrap" />
+            <TextBlock Text="{Binding }" Foreground="{DynamicResource SystemControlErrorTextForegroundBrush}" TextWrapping="Wrap" />
```

Comment:
I believe we should have converter from "object error" to "string". And in case of Exception, return Message from it.

Reply:
I think it will just work because, most frameworks just use string, the default theme does this too, and for exception, it will call ToString().

Reply:
It will be Type+Message